rapid7/metasploit-framework
# Prepare for migrating wiki to framework

Adds additional prep for migrating the existing wiki into the framework repo.
- Updates deprecation notice to link to the new doc location, as well as a link to the wiki's history in case the file goes AWOL in the future
- Adds a new `--create-wiki-to-framework-migration-branch` flag which will become the final PR to move the wiki into framework

## Verification

- Ensure CI passes
- Code review
- Verify `bundle exec ruby build.rb --create_wiki_to_framework_migration_branch` creates a new local `merge-metasploit-framework-wiki-into-metasploit-framework` branch successfully
- Verify `bundle exec ruby build.rb --update-wiki-deprecation-notice` generates the required deprecation notice changes
